### ISSUE TITLE
File upload PHP warning: strpos() expects parameter 1 to be string, object given

### ISSUE THREAD

**Issue by JbalTero** [directus/api#1154]
# Bug Report

Using api v2.3.0, I did the uploading of `/_/files`. It went successfully but there's a php warning saying: `strpos() expects parameter 1 to be string, object given` when env is development. This does not happen if set to production.

## Steps to Reproduce
* Fresh install of v2.3.0
* Set the `api.php` `env` to `development`
* Upload a file [Using Base64 Content](https://docs.directus.io/api/reference.html#using-base64-content)

## Expected Behavior

There should be no PHP warning

## Actual Behavior

Response: 
```json
{
  "data": {
    "id": 5,
    "storage": "local",
    "filename": "5958a71a-bdc0-5d89-8150-5fd15e73f369-5.jpg",
    "title": "5958a71a Bdc0 5d89 8150 5fd15e73f369 5",
    "type": "image\/jpeg",
    "uploaded_by": 1,
    "uploaded_on": "2019-07-27T14:33:30+00:00",
    "charset": "binary",
    "filesize": 9625,
    "width": 200,
    "height": 200,
    "duration": null,
    "embed": null,
    "folder": null,
    "description": "",
    "location": "",
    "tags": [],
    "checksum": "bd6cc4d714c1bd46accf243f58e6f8ed",
    "metadata": null,
    "data": {
      "full_url": "http:\/\/redacted\/uploads\/_\/originals\/5958a71a-bdc0-5d89-8150-5fd15e73f369-5.jpg",
      "url": "\/uploads\/_\/originals\/5958a71a-bdc0-5d89-8150-5fd15e73f369-5.jpg",
      "thumbnails": [
        {
          "url": "http:\/\/redacted\/thumbnail\/_\/200\/200\/crop\/good\/5958a71a-bdc0-5d89-8150-5fd15e73f369-5.jpg",
          "relative_url": "\/thumbnail\/_\/200\/200\/crop\/good\/5958a71a-bdc0-5d89-8150-5fd15e73f369-5.jpg",
          "dimension": "200x200",
          "width": 200,
          "height": 200
        }
      ],
      "embed": null
    }
  }
}<br/><b>Warning</b>: strpos()expectsparameter1tobestring,
objectgivenin<b>/redacted/src/core/Directus/Filesystem/Files.php</b>online<b>693</b><br/><br/><b>Warning</b>: strpos()expectsparameter1tobestring,
objectgivenin<b>/redacted/src/core/Directus/Filesystem/Files.php</b>online<b>238</b><br/><br/><b>Warning</b>: base64_decode()expectsparameter1tobestring,
objectgivenin<b>/redacted/src/core/Directus/Filesystem/Files.php</b>online<b>243</b><br/><br/><b>Warning</b>: strlen()expectsparameter1tobestring,
objectgivenin<b>/redacted/src/core/Directus/Filesystem/Files.php</b>online<b>275</b><br/><br/><b>Warning</b>: strlen()expectsparameter1tobestring,
objectgivenin<b>/redacted/src/core/Directus/Filesystem/Files.php</b>online<b>277</b><br/>
```

## Technical Details

- Device: Desktop
- OS: Win 10  10.0.18362.145
- Web Server: Apache/2.4.29
- PHP Version: PHP 7.1.30
- Database: 10.1.40
- Install Method: Downloaded [build release](https://github.com/directus/api/releases/download/v2.3.0/api-build-v2.3.0.zip)

**Comment by bjgajjar** [comment]
Hey @JbalTero 

This is a known issue and it's already fixed in #1134. So it will be resolved in the next release. :)

**Comment by JbalTero** [comment]
thanks
